app/scripts/controllers/permissions/specifications.js

```diff
@@ -11,7 +11,7 @@ import {
 /**
  * This file contains the specifications of the permissions and caveats
  * that are recognized by our permission system. See the PermissionController
- * README in @metamask/snap-controllers for details.
+ * README in @metamask/controllers for details.
```

Comment:
```suggestion
 * README in @metamask/permission-controller for details.
```

Reply:
Will update when we use the new packages